SECP-service/src/main/java/com/visucius/secp/resources/UserResource.java

```diff
@@ -98,4 +98,17 @@ public Response verifyDevice(@Auth @PathParam("name") String deviceName, @PathPa
     public Response getUsersPublicKeys(@Auth @PathParam("id") long id) {
         return userController.getUsersPublicKeys(id);
     }
+
+    @GET
+    @UnitOfWork
+    public Response getAllUsers() {
+        return userController.getAllUsers();
```

Comment:
Why do you need to get all the users?

Reply:
As stated in the previous PR, admins need to know all the active users so they can modify the roles as they see fit. run the app to see the changes that was made on the client side.

Reply:
Then you shouldn't return all the user's at once

Reply:
Why not?

Reply:
I mean you should return like 50 at time instead of returning all the users because if we have thousands of users the request will take to long

Reply:
I see your point and I agree. Do you mind implementing it for both the users and the groups just like how you did it for messages. Thanks